tsuru/client/jobs.go

```diff
@@ -142,14 +150,15 @@ func (c *JobCreate) Run(ctx *cmd.Context, cli *cmd.Client) error {
 		return err
 	}
 	j := tsuru.InputJob{
-		Name:        jobName,
-		Tags:        c.tags,
-		Schedule:    c.schedule,
-		Plan:        c.plan,
-		Pool:        c.pool,
-		Description: c.description,
-		TeamOwner:   c.teamOwner,
-		Manual:      c.manual,
+		Name:                  jobName,
+		Tags:                  c.tags,
+		Schedule:              c.schedule,
+		Plan:                  c.plan,
+		Pool:                  c.pool,
+		Description:           c.description,
+		TeamOwner:             c.teamOwner,
+		Manual:                c.manual,
+		ActiveDeadlineSeconds: c.maxRunningTime,
```

Comment:
@morpheu sending ActiveDeadlineSeconds as 0 is a problem?

Reply:
It should be handle on the API also, but on client I think we should send 0 to inform server that should set default value